freetextresponse/locale/vi/LC_MESSAGES/django.po

```diff
@@ -1,48 +0,0 @@
-# Vietnamese Translations for Free Text Response Xblock
-# plural translations are copies of the singular version
-
-# Copyright (C) 2017
-# This file is distributed under the same license as the free-text-response package.
-# Hang Pham <hpham3@stanford.edu>, 2017
-# 
-msgid ""
-msgstr ""
-"Project-Id-Version: 0.1.2\n"
```

Comment:
Why removing `Vietnamese`?

Reply:
I think the file is just being renamed to `text.po`?